src/actions/images.ts

```
@@ -327,10 +334,17 @@
   return injections;
 };
 
-const generateImages = async (context: MulmoStudioContext, settings?: Record<string, string>, callbacks?: CallbackFunction[]) => {
-  const options = await graphOption(context, settings);
+type ImageOptions = {
+  imageAgents: Record<string, unknown>,
+};
+const generateImages = async (context: MulmoStudioContext, settings?: Record<string, string>, callbacks?: CallbackFunction[], options?: ImageOptions) => {
+  const imageAgents = options?.imageAgents ?? {};
```

ESLint

[error] 'imageAgents' is already declared in the upper scope on line 27 column 7.

(no-shadow)